ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-isis-new.yml

```diff
@@ -9,6 +9,7 @@ router_isis:
   spf_interval:
     interval: 250
     wait_interval: 10
+    hold_interval: 20
```

Comment:
Do we not specify the units  here? Looks like EOS supports both with and without unit configs
```
coi418(config-router-isis)#spf-interval 250 10 20                                                                                                      
coi418(config-router-isis)#sh act
router isis evpn_underlay
   spf-interval 250 10 20 
```
and 
```
coi418(config-router-isis)#spf-interval 130 milliseconds 10 milliseconds 20 milliseconds 
coi418(config-router-isis)#sh act                                          
router isis evpn_underlay                                                  
   spf-interval 130 milliseconds 10 milliseconds 20 milliseconds   
```

Reply:
For me its invalid-
```
s1-leaf2(config-router-isis)#spf-interval 130 milliseconds 10 milliseconds 20 milliseconds 
% Invalid input
```

Which EOS version are you checking in?

Reply:
Hi @Shivani-gslab and @sugetha24 
There is two case on EOS(4.32.0F-35434449.eostrunkshipping):
1. If we give the unit for first timer then we have to provide the unit for other two timers also.
```
wan1-dc1(config-router-isis)#spf-interval 10 ?
  milliseconds  Configure maximum interval between two SPFs in milliseconds
  seconds       Configure maximum interval between two SPFs in seconds
  <1-300000>    Initial wait interval for SPF in milliseconds
  <cr>          

wan1-dc1(config-router-isis)#spf-interval 10 seconds ?
  <1-300000>  Initial wait interval for SPF in milliseconds
  <cr>        

wan1-dc1(config-router-isis)#spf-interval 10 seconds 20 ?
  milliseconds  Configure initial wait interval for SPF in milliseconds

wan1-dc1(config-router-isis)#spf-interval 10 seconds 20 milliseconds ?
  <1-300000>  Hold interval between the first and second SPF runs in milliseconds
  <cr>        

wan1-dc1(config-router-isis)#spf-interval 10 seconds 20 milliseconds 30 ?
  milliseconds  Configure hold interval between the first and second SPF runs in milliseconds

wan1-dc1(config-router-isis)#spf-interval 10 seconds 20 milliseconds 30 milliseconds ?
  <cr>  

wan1-dc1(config-router-isis)#spf-interval 10 seconds 20 milliseconds 30 milliseconds
```
2. If we dont give the unit for first timer then by default it takes seconds as unit and for other two we cant provide the unit but it supports with milliseconds(default) only
```
wan1-dc1(config-router-isis)#spf-interval 10 20 ?
  <1-300000>  Hold interval between the first and second SPF runs in milliseconds
  <cr>        

wan1-dc1(config-router-isis)#spf-interval 10 20 
```

Reply:
Added `interval_unit` key and implemented in template.